Integrations/RiskSense/RiskSense.py

```diff
@@ -0,0 +1,1975 @@
+
+    app_description = app_detail_dict.get('description', '')
+    if app_description:
+        hr += '\n ### Application Description:' + '\n' + app_description
+
+    return hr
+
+
+def get_request_timeout():
+    """
+    Validate and return the request timeout parameter.
+    The parameter must be a positive integer.
+    Default value is set to 60 seconds for API request timeout.
+
+    :params req_timeout: Request timeout value.
+    :return: boolean
+    """
+    try:
+        request_timeout = int(demisto.params().get('request_timeout'))
+        if request_timeout <= 0:
+            raise ValueError
+        return request_timeout
+    except ValueError:
+        return_error('HTTP Request Timeout parameter must be a positive integer.')
+
+
+def get_self_link(resp):
+    """
+    Retrieve self link from response.
+
+    :param resp: JSON response
+    :return: self link
+    """
+    return resp.get('_links', {}).get('self', {}).get('href', '')
+
+
+def fetch_page_details(resp):
+    """
+    Parse total element, page number and total pages from the page.
+
+    :param resp: json response.
+    :return: page details.
+    """
+    total_element = resp.get('page', {}).get('totalElements', 0)
+    page_number = resp.get('page', {}).get('number', 0)
+    total_pages = resp.get('page', {}).get('totalPages', 0)
+
+    return page_number, total_element, total_pages
+
+
+''' REQUESTS FUNCTIONS '''
+
+
+def test_module(client):
+    """
+        Performs basic GET request
+    :param client:
+    :return: None
+    """
+    client_name = demisto.params().get('client_name', '-')
+    resp_json = client.http_request('GET', url_suffix='')
+    clients = resp_json.get('_embedded', {}).get('clients', [])
+
+    # Verifying client name mentioned in integration configuration
+    if not any(client_info.get('name', '') == client_name for client_info in clients):
+        raise ValueError('Invalid client name configured.')
+    demisto.results('ok')
+
+
+def get_hosts_command(client, args):
+    """
+    Retrieve information about host(s) based on arguments.
+    Perform sorting based on argument. By default, it will sort by 'rs3' field.
+
+    :param client: client object
+    :param args: filter criteria provided by user.
+    :return: standard output.
+    """
+
+    # gather client detail from integration context
+    client_detail = get_client_detail_from_context(client)
+    client_id = client_detail['Id']
+    client_name = client_detail['ClientName']
+    url_suffix = '/' + str(client_id) + '/host/search'
+
+    # Prepares filter request body
+    data = prepare_filter_payload(args, 'detail')
+
+    resp = client.http_request('POST', url_suffix=url_suffix, json_data=data)
+    page_number, total_element, total_pages = fetch_page_details(resp)
+
+    if total_element == 0:
+        return 'No host(s) found for the given argument.', {}, {}
+    if page_number >= total_pages:
+        raise ValueError('Invalid page navigation.')
```

Comment:
This will end more gracefully this way
```suggestion
       return_error('Invalid page navigation.')
```

Reply:
@ronykoz  we have made given changes in entire source code for above ValueError exception.